tsl/src/continuous_aggs/common.c

```diff
@@ -0,0 +1,1563 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+#include "common.h"
+
+#define DEFAULT_MATPARTCOLUMN_NAME "time_partition_col"
+
+#define PRINT_MATCOLNAME(colbuf, type, original_query_resno, colno)                                \
+	do                                                                                             \
+	{                                                                                              \
+		int ret = snprintf(colbuf, NAMEDATALEN, "%s_%d_%d", type, original_query_resno, colno);    \
+		if (ret < 0 || ret >= NAMEDATALEN)                                                         \
+			ereport(ERROR,                                                                         \
+					(errcode(ERRCODE_INTERNAL_ERROR),                                              \
+					 errmsg("bad materialization table column name")));                            \
+	} while (0);
```

Comment:
What about transform those macro functions into `static inline`?? also changing this horrible name to something like `makeMaterializeColumnName`